src/lib/components/rehearsals/SongList.svelte

```diff
@@ -2,7 +2,7 @@
 	import { page } from '$app/stores'
 	import ScrollableList from '$lib/components/scrollableList.svelte'
 	import type { rehearsalSong } from '$lib/types/domain/rehearsal'
-	import type { song } from '$lib/types/domain/song'
+	import type { Song } from '$lib/types/domain/song'
```

Comment:
You know what, imma just go ahead and say it. screw boyscouting. Ik wil dit in een aparte merge gedaan hebben zodat t niet deze merge onleesbaar maakt

Reply:
voor nu fine, maar volgende keer pls

Reply:
Sawry, is gucci